shared/logutil/logutil.go

```diff
@@ -0,0 +1,83 @@
+package logutil
+
+import (
+	"os"
+	"strings"
+
+	joonix "github.com/joonix/log"
+	"github.com/sirupsen/logrus"
+	prefixed "github.com/x-cray/logrus-prefixed-formatter"
+)
+
+// WriterHook is a hook that writes logs of specified LogLevels to specified Writer
+type WriterHook struct {
+	LogLevels []logrus.Level
+}
+
+// Fire will be called when some logging function is called with current hook
+// It will format log entry to string and write it to appropriate writer
+func (hook *WriterHook) Fire(entry *logrus.Entry) error {
+	line, err := entry.String()
+	if err != nil {
+		return err
+	}
+	//simply call the file logger Println func after removing the new line char
+	line = strings.TrimSuffix(line, "\n")
+	fileLogger.Println(line)
+	return err
+}
+
+// Levels define on which log levels this hook would trigger
+func (hook *WriterHook) Levels() []logrus.Level {
+	return hook.LogLevels
+}
+
+// File Logger insyance
+var fileLogger = &logrus.Logger{
+	Level: logrus.TraceLevel,
+}
+
+//ConfigurePersistentLogging starts a persistent file logger
+func ConfigurePersistentLogging(logFileName string, logFileFormatName string) (bool, error) {
```

Comment:
Why does this return a bool?

```suggestion
func ConfigurePersistentLogging(logFileName string, logFileFormatName string) error {
```